libs/kotaemon/kotaemon/loaders/azureai_document_intelligence_loader.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+from typing import Optional
+
+from kotaemon.base import Document, Param
+
+from .base import BaseReader
+
+
+class AzureAIDocumentIntelligenceLoader(BaseReader):
+    """Utilize Azure AI Document Intelligence to parse document
+
+    As of April 24, the supported file formats are: pdf, jpeg/jpg, png, bmp, tiff,
+    heif, docx, xlsx, pptx and html.
+    """
+
+    _dependencies = ["azure-ai-documentintelligence"]
+
+    endpoint: str = Param("Endpoint of Azure AI Document Intelligence")
```

Comment:
should we add default credential from env var (e.g: AZURE_FORM_RECOGNIZER_KEY, AZURE_FORM_RECOGNIZER_ENDPOINT) for ease of configuration?

Reply:
@taprosoft it's possible. Let me add this.